extensions/jni/deployment/src/main/java/io/quarkus/jni/deployment/JniProcessor.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.jni.deployment;
+
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.JniBuildItem;
+
+public class JniProcessor {
+
+    @BuildStep
+    void setupJni(JniBuildItem jniBuildItem) {
```

Comment:
This should not have its own extension. If a user wants to enable JNI they should be able to do it through application.properties. If an extension needs it they should be able to just produce a build item.

Reply:
Ok so I move Processor to the core

Reply:
done